src_bonus/main.cpp

```diff
@@ -0,0 +1,41 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   main.cpp                                           :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: mcourtoi <mcourtoi@student.42.fr>          +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2024/03/13 13:13:11 by mcourtoi          #+#    #+#             */
+/*   Updated: 2024/03/14 18:56:14 by mcourtoi         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "class/Bot.hpp"
+#include <iostream>
+#include <cstdlib>
+
+int main(int const ac, char const *const *const av)
+{
+	if (ac < 3)
+	{
+		std::cerr << "Usage: " << av[0] << " <port> <password>" << std::endl;
+		return 1;
+	}
+
+	int         port = atoi(av[1]);
```

Comment:
`port` can be `const`, and thus, should be + you don't handle possible int overflows.